src/config/dynamic-theme-fixes.config

```diff
@@ -1080,6 +1080,14 @@ CSS
 
 ================================
 
+aprsdirect.com
+
+INVERT
+.leaflet-tile-pane
+img[src^="https://maps.googleapis.com/maps/"]
```

Comment:
I couldn't find such element, and would also probably be conflicting with the `leaflet-tile-pane`.